src/WebJobs.Script/Workers/Rpc/RpcWorkerConstants.cs

```diff
@@ -51,5 +51,12 @@ public static class RpcWorkerConstants
         public const string DotNetExecutableName = "dotnet";
         public const string DotNetExecutableNameWithExtension = DotNetExecutableName + ".exe";
         public const string DotNetFolderName = "dotnet";
+
+        // Concurrancy settings
+        public const string PythonTreadpoolThreadCount = "PYTHON_THREADPOOL_THREAD_COUNT";
+        public const string PSWorkerInProcConcurrencyUpperBound = "PSWorkerInProcConcurrencyUpperBound";
+        public const string DefaultConcurrecnyPython = "32";
+        public const string DefaultConcurrencyPS = "100";
```

Comment:
I really doubt we need separate values for Python and PowerShell. Why would we assume Python can handle less threads than PowerShell? I think the main idea of what we are currently doing about language worker concurrency is that we want to stop _assuming without evidence_. If the latency is not growing, we are not running out of memory, etc., let's keep increasing concurrency, because why not? Until we start executing functions, we simply don't have data for meaningful decisions per language, and we should stop pretending that we do. The limit here should not be something that we carefully design, this is just a very large number for an extra sanity check. So, I would suggest we keep a single number and increase it even more aggressively: 1000 or 10,000.

Reply:
For python 3.8 and it takes max concurrency=32. I tried 33 and the concurrency resets to 1. I do not know who limits the concurrency python worker or python runtime. Need to chat with Varad/Roger about this.

For PS I will change it to 1000.

Reply:
1. There was a bug in the Python worker, sync up with Varad about current max concurrency.
2. Most likely in future the host would be sending "concurrency capability" for the worker and the worker will set the internal concurrency himself.

After discussing offline @AnatoliB we decided to leave it until the workers change.

Reply:
The fix is not in Python worker yet so we want to go with DefaultConcurrecnyPython=32 for now.